### DISCUSSION TITLE
Check Elastalert2 version (docker container)

### DISCUSSION THREAD

**Discussion by smith-jr** [jertel/elastalert2#1139] [Q&A] [answered]
Hi! I'm using elastalert in docker container. How can I check its version? Is there any command for this?

**Comment by jertel** [comment, marked as answer]
If you're using the official ElastAlert 2 image created by this project then the version number will be in the image tag, if you are pinning to specific versions. If you're using the `latest` or `2` tag then you will need to compare the image hash to the hashes listed on either [hub.docker.com](https://hub.docker.com/r/jertel/elastalert2/tags) or [ghcr.io](https://github.com/jertel/elastalert2/pkgs/container/elastalert2%2Felastalert2), depending on which registry you use.

**Reply by smith-jr** [reply]
thanks @jertel
